radixdlt-cli/src/main/groovy/com/radixdlt/cli/ValidatorKeyGenerator.groovy

```diff
@@ -0,0 +1,49 @@
+package com.radixdlt.cli
+
+import com.radixdlt.crypto.ECKeyPair
+import com.radixdlt.crypto.RadixKeyStore
+import picocli.CommandLine
+
+@CommandLine.Command(name = "generate-validator-key",mixinStandardHelpOptions = true,

```

Comment:
Would be better if we included some Javadoc here about how to run the command line applet.

Reply:
Also might be worth printing out the public key once complete.  Maybe replace the "Done" message with something like "Key AAkasjdfl3h4/12 written to keystore.ks", or changing the initial message to "Writing key asdfjhasdfl into new keystore keystore.ks".

Reply:
OK